CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.11.0 (unreleased)
+
+**Breaking Changes**
+
+### Stack ids can no longer contain whitespaces
+
+A `TerraformStack` may no longer contain whitespace characters, since we rely on paths being whitespace free. If you have a stack with an id containing a whitespace, please replace it with a hyphen.
```

Comment:
Sorry for missing this in my first review, but renaming a stack could change the name of the default local backend state file, right? I feel like we should warn users that they might lose state (maybe link them to some docs about renaming stacks – if we have such 😅)

Reply:
No good point, I entirely missed that